route53_static.tf

```diff
@@ -33,6 +33,75 @@ resource "aws_ses_domain_dkim" "cyber_dhs_gov_dkim" {
   domain = aws_ses_domain_identity.cyhy_dhs_gov_identity.domain
 }
 
```

Comment:
I'd like to see a `----\n stuff \n -----` header here separating the SNS stuff from the DNS.  It could go to its own file as well.  I'm fine either way.  I think a new repo is overkill.  This is pretty tightly coupled to what we do here with identity management.

Reply:
I will add the comment.

If I created another repo then I would move the `aws_ses_domain_identity` resources to the new repo as well.

Reply:
See commit f6f1793.